recipes/ecfeed/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.18)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.15)
```

do not need 3.18 feature. 3.15 will be the default version for Conan 2.0

Reply:
Updated.